### PR TITLE
days to hours in event que

### LINKED ISSUE #56: Incompatible with Upcoming Events mod
We spawn our custom event queue items so that mod's overrides do nothing. However there is no conflict either

### PR DESCRIPTION
**Pull request by statusNone**
Closes #56

### CODE REVIEW

File: CovertInfiltration/Src/CovertInfiltration/Classes/UIListener_ModConfigMenu.uc [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	LoadSavedSettings();
+
+	Page = ConfigAPI.NewSettingsPage("Covert Infiltration");
+	Page.SetPageTitle("Covert Infiltration");
+	Page.SetSaveHandler(SaveButtonClicked);
+
+	Group = Page.AddGroup('Group1', "Various Settings");
+
+	Group.AddCheckBox('checkbox', "Hours instead of days", "Display hours instead of days in the EventQueue", DAYS_TO_HOURS, CheckboxSaveHandler);
+	Group.AddSlider('slider', "Days before hours", "How many days left before displaying hours instead", 1, 3, 1, DAYS_BEFORE_HOURS, SliderSaveHandler);
```
Comment (robojumper) [inline CovertInfiltration/Src/CovertInfiltration/Classes/UIListener_ModConfigMenu.uc]: All of this needs `localized` strings.

File: CovertInfiltration/Src/CovertInfiltration/Classes/UIEventQueue_CovertActionListItem_CI.uc [hunk 2]
```diff
@@ -7,12 +7,50 @@
...
+	local string TimeValue, TimeLabel, Desc;
+	
+	local bool DaysToHours;
+	local int DaysBeforeHours;
+
+	Desc = Event.Data;
 	ActionRef = Event.ActionRef;
+
+	DaysToHours = `MCM_CH_GetValue(class'ModConfigMenu_Defaults'.default.DAYS_TO_HOURS_DEFAULT, class'UIListener_ModConfigMenu'.default.DAYS_TO_HOURS);
+	DaysBeforeHours = `MCM_CH_GetValue(class'ModConfigMenu_Defaults'.default.DAYS_BEFORE_HOURS_DEFAULT, class'UIListener_ModConfigMenu'.default.DAYS_BEFORE_HOURS);;
```
Comment (Xymanek) [inline CovertInfiltration/Src/CovertInfiltration/Classes/UIEventQueue_CovertActionListItem_CI.uc]: ;;

File: CovertInfiltration/Src/CovertInfiltration/Classes/UIEventQueue_ListItem_CI.uc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+simulated function UIEventQueue_ListItem InitListItem()
+{
+	super.InitPanel(); // must do this before adding children or setting data
+	return self; 
+}
+
+simulated function OnInit()
+{
+	super.OnInit();
+}
```
Comment (Xymanek) [inline CovertInfiltration/Src/CovertInfiltration/Classes/UIEventQueue_ListItem_CI.uc]: Redundant function override, remove

File: CovertInfiltration/Src/CovertInfiltration/Classes/UIEventQueue_ListItem_CI.uc [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+	{
+		TimeLabel = class'UIUtilities_Text'.static.GetColoredText(TimeLabel, eUIState_Cash);
+		TimeValue = class'UIUtilities_Text'.static.GetColoredText(TimeValue, eUIState_Cash);
+	}
+
+	SetTitle(Desc);
+	SetDaysLabel(TimeLabel);
+	SetDaysValue(TimeValue);
+	SetIconImage(Event.ImagePath);
+
```
Comment (Xymanek) [inline CovertInfiltration/Src/CovertInfiltration/Classes/UIEventQueue_ListItem_CI.uc]: Remove empty line

File: CovertInfiltration/Src/CovertInfiltration/Classes/UIListener_ModConfigMenu.uc [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+simulated function SaveButtonClicked(MCM_API_SettingsPage Page)
+{
+    self.CONFIG_VERSION = `MCM_CH_GetCompositeVersion();
+    self.SaveConfig();
+}
+
+defaultproperties
+{
+	ScreenClass=none
+}
```
Comment (Xymanek) [inline CovertInfiltration/Src/CovertInfiltration/Classes/UIListener_ModConfigMenu.uc]: Redundant block, remove

**Comment by Xymanek** [comment]
Rebase please

File: CovertInfiltration/Src/CovertInfiltration/Classes/ModConfigMenu_Defaults.uc [hunk 6]
```diff
@@ -0,0 +1,9 @@
+//---------------------------------------------------------------------------------------
+//  ModConfigMenu Defaults
+//---------------------------------------------------------------------------------------
+class ModConfigMenu_Defaults extends Object config(ModConfigMenu_Defaults);
```
Comment (Xymanek) [inline CovertInfiltration/Src/CovertInfiltration/Classes/ModConfigMenu_Defaults.uc]: Please use correct header style

File: CovertInfiltration/Src/CovertInfiltration/Classes/UIEventQueue_ListItem_CI.uc [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+class UIEventQueue_ListItem_CI extends UIEventQueue_ListItem;
```
Comment (Xymanek) [inline CovertInfiltration/Src/CovertInfiltration/Classes/UIEventQueue_ListItem_CI.uc]: Please add doc header

**Pull request merged**